.github/workflows/ci.yml

```diff
@@ -31,10 +31,11 @@ jobs:
         name: test-reports
         path: build/reports/tests
     - name: Release
-      if: github.event.inputs.release == 'yes'
+      if: github.event.inputs.release != 'None'
       env:
         atlassian_private_username: ${{ secrets.ARTIFACTORY_USERNAME }}
         atlassian_private_password: ${{ secrets.ARTIFACTORY_API_KEY }}
       run: |
+        ./gradlew markNextVersion -Prelease.incrementer=increment$ {{ github.event.inputs.release }} -Prelease.localOnly
```

Comment:
Does it work?

Reply:
It worked locally for me. We'll know the full flow if we actually release 1.1.0.
@tciesla-atlassian wanna try or should I test my own contraptions? 😁

Reply:
Tomek bravely tested it and we got [a green 1.1.0 release](https://github.com/atlassian-labs/db-replica/runs/2140369424) 😎